doc/nrf/known_issues.rst

```diff
@@ -430,7 +443,7 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-8-0
+.. rst-class:: v1-9-0 v1-8-0
 
 KRKNWK-11720: NCP communication gets stuck
   The communication between the SoC and the NCP Host sometimes stops on the SoC side.
```

Comment:
This one is still valid, but this list has a different JIRA number for it... https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Release+notes+and+known+issues+1.9.0

Not sure what the status is. @maciejpietras ?

Reply:
11720 was closed as duplication and is continued in 11704

Reply:
Updated the JIRA number for this issue to 11704.